src/components/CodeView/index.tsx

```diff
@@ -64,6 +64,10 @@ export const scrollToSelectedLine = (element: HTMLElement | null) => {
   }
 };
 
+export const trimCode = (content: string) => {
+  return `${content.substring(0, TRIMMED_CHAR_COUNT)} ${contentAddedByTrimmer}`;
+};
```

Comment:
if `codeShouldBeTrimmed` is in utils.py, should this function be too?

Reply:
We have different logic for trimming code in the browse view and the diff view, so each component (`CodeView` and `DiffView`) contains its own method for doing this. I think that makes it easier to understand than moving them both to `utils` (`.tsx`, btw, not `.py`, 😉 ) and giving them unique names there. It also means that this is only ever called once, from here, so moving it to `utils` doesn't really yield a benefit.